src/store/v3MigrationSelectors.ts

```diff
@@ -83,6 +83,7 @@ export type V3Rates = {
   aIncentivesData?: ReserveIncentiveResponse[];
   vIncentivesData?: ReserveIncentiveResponse[];
   sIncentivesData?: ReserveIncentiveResponse[];
+  priceInUSD: string;
```

Comment:
i'm not sure this type belongs here, but i didn't want to add more unwrapped props